src/compiler/diagnosticMessages.json

```diff
@@ -2529,6 +2529,14 @@
         "category": "Error",
         "code": 2742
     },
+    "No overload expects {0} type argument(s), but an overload does exist that expects {1} type argument(s).": {
+        "category": "Error",
+        "code": 2743
+    },
+    "No overload expects {0} type argument(s), but overloads do exist that expect either {1} or {2} type argument(s).": {

```

Comment:
This wording is a lot easier to scan: "Expected at least {0} or at most {1} type arguments, but got {2}."

It overpromises a bit &mdash; "at least {0}" arguments is not actually guaranteed to accept less than {0} arguments &mdash; but it's much easier to read. If the programmer overshoots, then they will get a follow-up message correcting them again.

Reply:
Never mind. See below.